things/postgres/things_test.go

```diff
@@ -605,6 +613,14 @@ func TestMultiThingRetrievalByChannel(t *testing.T) {
 			},
 			size: n - thsDisconNum,
 		},
+		"retrieve all things by channel without limit for existing owner": {
```

Comment:
Do we have `retrieve all non connected things by channel without limit` test?  BTW, why with existing owner? Not sure that it's consitent with other test titles